services/frontend-v3/src/App.jsx

```diff
@@ -75,33 +70,23 @@ const App = () => {
       messages={i18nConfig.messages}
       formats={i18nConfig.formats}
     >
-      <Router history={historySingleton}>
-        <Switch>
-          <Route exact path="/" render={() => <LandingPage />} />
-          <Route
-            path="/secured"
-            render={({ location }) => <Secured location={location} />}
-          />
-          <Route path="/about" render={() => <About type="about" />} />
-          <Route path="/help" render={() => <About type="help" />} />
-          <Route path="/terms" render={() => <About type="terms" />} />
-          <Route path="/privacy" render={() => <About type="privacy" />} />
-          <Route path="/admin" render={() => <Admin />} />
-          <Route path="/error" render={() => <UnexpectedError />} />
-          <Route path="/forbidden" render={() => <Forbidden />} />
-          <Route render={() => <NotFound />} />
-        </Switch>
-      </Router>
+      <Routes />
     </IntlProvider>
   );
 };
 
 const ReduxWrappedApp = () => (
-  <ReduxProvider store={store}>
-    <PersistGate loading={null} persistor={persistor}>
-      <App />
-    </PersistGate>
-  </ReduxProvider>
+  <KeycloakProvider
+    keycloak={keycloak}
+    initConfig={keycloakConfig}
+    LoadingComponent={() => <AppLayout loading />}
+  >
+    <ReduxProvider store={store}>
+      <PersistGate loading={null} persistor={persistor}>
+        <App />
+      </PersistGate>
+    </ReduxProvider>
+  </KeycloakProvider>
 );
```

Comment:
Could probably add the `<IntlProvider />` to this, and should probably rename the const to Provider or/and just create a new component that just wraps its children in another file (the second option is probably the cleanest. it would probably debloat the App.jsx